### PR TITLE
core API

### PR DESCRIPTION
**Pull request by awiouy**

### CODE REVIEW

**Comment by awiouy** [comment]
Yay! rustfmt in Travis works :)

File: core/src/util/mod.rs [hunk 1]
```diff
@@ -6,10 +6,9 @@ use std::mem;
 use std::ops::{Mul, Rem, Shr};
 
 mod int128;
-mod spotify_id;
 
+pub use spotify_id::{FileId, SpotifyId};
```
Comment (plietar) [inline core/src/util/mod.rs]: This is reexporting the two types as `core::FileId` and `core::SpotifyId`.
I don't think this line is needed at all.

**Comment by awiouy** [comment]
Thanks for the heads up @plietar 
`Session.spawn()` is used in `audio/src/fetch.rs` and can therefore be removed from #130

**Comment by sashahilton00** [comment]
Will merge this, followed by #150. With regards to meteadta.rs, is there anything stopping us just deleting that file?

**Pull request merged**